<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GrantsFundingGraphDash</class>
 <widget class="QDialog" name="GrantsFundingGraphDash">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCustomPlot" name="bar_graphGrants" native="true">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>3000</width>
       <height>3000</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchInGrants"/>
     </item>
     <item>
      <widget class="QComboBox" name="fromCBGrants"/>
     </item>
     <item>
      <widget class="QComboBox" name="toCBGrants"/>
     </item>
     <item>
      <widget class="QPushButton" name="graphBtn">
       <property name="text">
        <string>Bar Graph</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pieBtn">
       <property name="text">
        <string>Pie Graph</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Print Bar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Print Pie</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
